state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1365,6 +1481,20 @@ pub mod pallet {
 			Ok(())
 		}
 
+		pub fn trigger_commission_distribution() {
```

Comment:
I don't think we need this - it's not clear that automatic distribution is required, and there are annoying edge cases to consider (ie. do we send dust, what if the amount is lower than the fee, etc, etc).